django/db/models/fields/__init__.py

```diff
@@ -113,6 +113,8 @@ class Field(RegisterLookupMixin):
                              "%(date_field_label)s %(lookup_type)s."),
     }
     class_lookups = default_lookups.copy()
+    deprecated_for_migrations = False
```

Comment:
Good point. Using `None` seems to make more sense, now that the variables hold more information than just yes/now.